src/lib/sidebar/EntireScheme.svelte

```diff
@@ -55,12 +57,17 @@
     }
   }
 
+  function openClearAllDialogue() {
+    displayClearAllConfirmation = true;
+  }
+
+  function cancelClearAll() {
```

Comment:
As a more general code style question, when should we try and inline the event handler, and when should we use separate functions like this? I think I've been pretty inconsistent about it so far. I kind of like this style, because the function name can describe "what" instead of "how". Though "displayClearAllConfirmation = false" is also pretty self-descriptive

Reply:
As a general rule my preference is towards more lines/words if it makes it more readable. But you're right that here it doesn't necessarily do that.  

The only other thing I was thinking was that we might want to eventually to do some other cleanup at the same time, but I can't think what that would be so don't really feel too strongly either way in this case.

Reply:
More explicitly readable SGTM. If you find other parts of the code where things are too terse / acronymish, let's clean it up!